Switching vesting org to registrar user doesn't work in admin console.

Error:

```
Internal Server Error: /admin/perma/linkuser/946/
Traceback (most recent call last):
  File "/srv/www/perma-envs/perma/lib/python2.7/site-packages/django/core/handlers/base.py", line 111, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/srv/www/perma-envs/perma/lib/python2.7/site-packages/django/contrib/admin/options.py", line 583, in wrapper
    return self.admin_site.admin_view(view)(*args, **kwargs)
  File "/srv/www/perma-envs/perma/lib/python2.7/site-packages/django/utils/decorators.py", line 105, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "/srv/www/perma-envs/perma/lib/python2.7/site-packages/django/views/decorators/cache.py", line 52, in _wrapped_view_func
    response = view_func(request, *args, **kwargs)
  File "/srv/www/perma-envs/perma/lib/python2.7/site-packages/django/contrib/admin/sites.py", line 206, in inner
    return view(request, *args, **kwargs)
  File "/srv/www/perma-envs/perma/lib/python2.7/site-packages/django/contrib/admin/options.py", line 1456, in change_view
    return self.changeform_view(request, object_id, form_url, extra_context)
  File "/srv/www/perma-envs/perma/lib/python2.7/site-packages/django/utils/decorators.py", line 29, in _wrapper
    return bound_func(*args, **kwargs)
  File "/srv/www/perma-envs/perma/lib/python2.7/site-packages/django/utils/decorators.py", line 105, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "/srv/www/perma-envs/perma/lib/python2.7/site-packages/django/utils/decorators.py", line 25, in bound_func
    return func.__get__(self, type(self))(*args2, **kwargs2)
  File "/srv/www/perma-envs/perma/lib/python2.7/site-packages/django/db/transaction.py", line 394, in inner
    return func(*args, **kwargs)
  File "/srv/www/perma-envs/perma/lib/python2.7/site-packages/django/contrib/admin/options.py", line 1404, in changeform_view
    self.save_model(request, new_object, form, not add)
  File "/srv/www/perma-envs/perma/lib/python2.7/site-packages/django/contrib/admin/options.py", line 1045, in save_model
    obj.save()
  File "/srv/www/perma-envs/perma/lib/python2.7/site-packages/model_utils/tracker.py", line 134, in save
    ret = original_save(**kwargs)
  File "/srv/www/perma/perma_web/perma/models.py", line 196, in save
    assert not (bool(self.vesting_org.all()) and self.registrar_id), "Users cannot have both a registrar and a vesting org."
AssertionError: Users cannot have both a registrar and a vesting org.
```

--- a/perma_web/perma/models.py
+++ b/perma_web/perma/models.py
@@ -188,13 +188,6 @@ class LinkUser(AbstractBaseUser):
         verbose_name = 'User'
 
     def save(self, *args, **kwargs):
-        # Don't allow users to have both a registrar and vesting_org.
-        # This is just a data consistency check -- logic earlier in the app should make sure this doesn't happen.
-        # Our m2m relationship on linksuser/vestingorg forces us to find out if
-        # this is a new item (self.pk == None for new items)
-        if self.pk is not None:
-            assert not (bool(self.vesting_org.all()) and self.registrar_id), "Users cannot have both a registrar and a vesting org."
-
         super(LinkUser, self).save(*args, **kwargs)
 
         # make sure root folder is created for each user.

